proposals/custom-descriptors/Overview.md

```diff
@@ -570,111 +576,33 @@ subtype ::=
   | ct:sharecomptype => sub final eps ct
 ```
 
-### Exact Reference Types
+### Exact Types
 
-Rather than use two new opcodes in the type opcode space
-to represent nullable and non-nullable exact reference types,
-we introduce just a single new prefix opcode that encode both:
+Exact heap types are introduced with a prefix byte:
 
 ```
-reftype :: ...
-  | 0x62 0x64 ht:heaptype => ref exact ht
-  | 0x62 0x63 ht:heaptype => ref null exact ht
+heaptype :: ... | 0x62 x:u32 => exact x
```

Comment:
0x62 made sense as the next opcode in the `reftype` production. In the `heaptype` production the next available opcode is 0x69. But also we might want to leave the 0x6x range entirely available for future abstract heap types? Perhaps 0x5F would then start a new category for heap types prefixes?